docs/src/main/sphinx/installation/cli.rst

```diff
@@ -29,6 +29,23 @@ The CLI uses the HTTP protocol and the
 :doc:`Trino client REST API </develop/client-protocol>` to communicate
 with Trino.
 
+TLS/HTTPS
+---------
+
+A Trino server is typically available with a HTTPS URL, so all network traffic
```

Comment:
For American pronunciation, the phrase is "an HTTPS", where H is pronounced aitch.
For Irish and apparently Austrian pronunciation, the phrase is "a HTTPS", where H is pronounced haytch. 
(The decision to use a vs an is based solely on pronunciation of the following word, not spelling.)
Dare I suggest that a US-based company use US spelling based on US pronunciation?

Reply:
I have no preference or idea on what is actually correct. In school I learned "Oxford (British) English" but at this stage the lines are blurry.. also because Canada theoretically still uses British... 

updated to use "an" anyw